applications/nrf_desktop/configuration/nrf54l15pdk_nrf54l15_cpuapp/hid_keyboard_leds_def_keyboard.h

```diff
@@ -15,13 +15,22 @@
  */
 const struct {} hid_keyboard_leds_def_include_once;
 
+/* For nRF54L15 SoC, only GPIO1 port can be used for PWM hardware peripheral output.
+ * Because of that, when using an nRF54L15 PDK, the following limitations must be considered:
+ *
+ * - For v0.2.x onboard LED1 cannot be used for PWM output.
+ * - For v0.3.0 onboard LED0 and LED2 cannot be used for PWM output.
+ *
+ * You can still use these LEDs with PWM LED driver, but LED color must be set either to
+ * LED_COLOR(255, 255, 255) or LED_COLOR(0, 0, 0) to avoid using PWM peripheral for the LEDs.
+ */
```

Comment:
Try to use the same content everywhere for consistency
```suggestion
/* For nRF54L15 SoC, only GPIO1 port can be used for PWM hardware peripheral output.
 * Because of that, when using an nRF54L15 PDK, the following limitations must be considered:
 *
 * - For v0.2.1 onboard LED1 cannot be used for PWM output.
 * - For v0.3.0 onboard LED0 and LED2 cannot be used for PWM output.
 *
 * You can still use these LEDs with PWM LED driver, but LED color must be set either to
 * LED_COLOR(255, 255, 255) or LED_COLOR(0, 0, 0) to avoid using PWM peripheral for the LEDs.
 */
```